enforcer-rules/README.md

```diff
@@ -0,0 +1,66 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule runs [Linkage Checker](../dependencies) for Maven projects.
+The rule enforces that project does not have linkage errors in its class path.
+
+- If the rule runs within a BOM's pom.xml, it checks a class path consisting of the managed
+dependencies and their transitive dependencies.
+- Otherwise, the rule checks a class path consisting of direct dependencies and their transitive
+dependencies.
+
+# Usage
+
+Add following plugin configuration to your `pom.xml`.
+
+```xml
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M2</version>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.cloud.tools.opensource</groupId>
+            <artifactId>linkage-checker</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+          </dependency>
+        </dependencies>
+        <executions>
+          <execution>
+            <id>enforce</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <linkageCheckerRule
```

Comment:
rules seem to be named as verbs: https://maven.apache.org/enforcer/enforcer-rules/index.html

Maybe banLinkageErrors

Reply:
Updated.